app/components/Dashboard/SimpleDepositWithdraw.jsx

```diff
@@ -222,77 +215,9 @@ class DepositWithdrawContent extends DecimalChecker {
             this._checkBalance
         );
     }
-
-    _checkFeeStatus(account = this.props.sender) {
-        if (!account) return;
-
-        const assets = ["1.3.0", this.state.to_withdraw.asset_id];
-        let feeStatus = {};
-        let p = [];
-        assets.forEach(a => {
-            p.push(
-                checkFeeStatusAsync({
-                    accountID: account.get("id"),
-                    feeID: a,
-                    options: ["price_per_kbyte"],
-                    data: {
-                        type: "memo",
-                        content:
-                            this.props.backingCoinType.toLowerCase() +
-                            ":" +
-                            this.state.toAddress +
-                            (this.state.memo ? ":" + this.state.memo : "")
-                    }
-                })
-            );
-        });
-        Promise.all(p)
-            .then(status => {
-                assets.forEach((a, idx) => {
-                    feeStatus[a] = status[idx];
-                });
-                if (!utils.are_equal_shallow(this.state.feeStatus, feeStatus)) {
-                    this.setState({
-                        feeStatus
-                    });
-                }
-                this._checkBalance();
-            })
-            .catch(err => {
-                console.error(err);
-            });
-    }
-
-    _updateFee(fee_asset_id = this.state.fee_asset_id) {
-        if (!this.props.sender) return null;
-        checkFeeStatusAsync({
-            accountID: this.props.sender.get("id"),
-            feeID: fee_asset_id,
-            options: ["price_per_kbyte"],
-            data: {
-                type: "memo",
-                content:
-                    this.props.backingCoinType.toLowerCase() +
-                    ":" +
-                    this.state.toAddress +
-                    (this.state.memo ? ":" + this.state.memo : "")
-            }
-        }).then(({fee, hasBalance, hasPoolBalance}) => {
-            this.setState(
-                {
-                    feeAmount: fee,
-                    hasBalance,
-                    hasPoolBalance,
-                    error: !hasBalance || !hasPoolBalance
-                },
-                this._checkFeeStatus
-            );
-        });
-    }
-
     _getCurrentBalance() {
         let balances = this.props.balance
-            ? [this.props.balance]
+            ? [ChainStore.getObject(this.props.balance)]
```

Comment:
Why was this addeD?

Reply:
Because there is was an issue, the balance props contain the asset id only and  the balances props contains the scope of the assets